dart/lib/src/http_client/sentry_http_client.dart

```diff
@@ -33,10 +23,6 @@ import 'failed_request_client.dart';
 /// );
 /// ```
 ///
-/// It starts and finishes a Span if there's a transaction bound to the Scope
-/// through the [TracingClient] client, it's disabled by default.
-/// Set [networkTracing] to `true` to enable it.
```

Comment:
README still contains `networkTracing` usage

Reply:
`SentryDioClientAdapter` still also uses `recordBreadcrumbs` and `networkTracing`

Reply:
Sorry I have been a bit busy, I will complete it by tomorrow.